testgrid/config/config.yaml

```diff
@@ -2863,8 +2863,62 @@ dashboards:
 
 - name: sig-apps
   dashboard_tab:
-  - name: TODO
-    test_group_name: ci-kubernetes-e2e-gce-slow  # TODO(fejta): fix
+  - name: gce
```

Comment:
I think we may want the slow and serial tests as well represented here at a later stage. For now, the list looks ok.